frontend/src/test/kea-test-utils/functions/printActions.ts

```diff
@@ -1,13 +1,22 @@
 import { ExpectFunction, testUtilsContext } from '~/test/kea-test-utils'
 
+export interface PrintActionsOptions {
+    compact?: boolean
+}
+
 export const printActions: ExpectFunction<any> = {
-    common(logic, payload) {
+    common(logic, options: PrintActionsOptions) {
         const { recordedHistory, historyIndex } = testUtilsContext()
 
-        console.log(`💈 Logging actions for logic "${logic.pathString}": ${payload ?? ''}`)
+        const text = recordedHistory
+            .map(({ action }, index) => {
+                const icon = historyIndex === index ? '👉' : action.type.includes(`(${logic.pathString})`) ? '🥦' : '🐠'
```

Comment:
despite my appreciation for emoji... does 🥦  and 🐠 have significance here?

Reply:
Not really and we can definitely change this 🥦 
I just took something bold and something light.